src/app/core/core.component.ts

```diff
@@ -52,4 +54,37 @@ export class CoreComponent implements OnInit, OnDestroy {
   isSolicitor(): boolean {
     return this.profile.isSolicitor();
   }
+
+  isUnsupportedBrowser(): boolean {
```

Comment:
this method should not be on the core component. It would be better if we could move this into the ccd-browser-support.component itself

Reply:
The scope will not satisfy if this method is moved into ccd-browser-support.component